client/jetpack-connect/authorize.js

```diff
@@ -1173,19 +1171,12 @@ export class JetpackAuthorize extends Component {
 
 		return (
 			<LoggedOutFormLinks>
-				{ ! isJetpackMagicLinkSignUpFlow && this.renderBackToWpAdminLink() }
 				<LoggedOutFormLinkItem
 					href={ loginURL }
 					onClick={ ( e ) => this.handleSignIn( e, loginURL ) }
 				>
 					{ translate( 'Sign in as a different user' ) }
 				</LoggedOutFormLinkItem>
-				{ ! isJetpackMagicLinkSignUpFlow && (
-					<LoggedOutFormLinkItem onClick={ this.handleSignOut }>
-						{ translate( 'Create a new account' ) }
-					</LoggedOutFormLinkItem>
-				) }
-				{ ! isJetpackMagicLinkSignUpFlow && <HelpButton /> }
```

Comment:
The removal of these three links (back to WP Admin, Sign out, Help) affects the unit test snapshot (`authorize.js.snap`). The unit test ran with the `jetpack/magic-link-signup` flag being `false` but now the same condition is always `true`.

There's also no longer a need to mock `config.isEnabled` in that test, that explains the code removal in `test/authorize.js`.